metric_learn/itml.py

```diff
@@ -167,6 +181,17 @@ class ITML_Supervised(_BaseITML, TransformerMixin):
 
   Attributes
   ----------
+  bounds_ : `list` of two numbers
+      Bounds on similarity, aside slack variables, s.t. ``d(a, b) < pos`` for
+      all given pairs of similar points ``a`` and ``b``, and
+      ``d(c, d) > neg`` for all given pairs of dissimilar points ``c`` and
+      ``d``, with ``bounds=[pos, neg]``, and ``d`` the learned distance. If
+      not provided at initialization, these are the ones derived at train
+      time.
```

Comment:
I am not familiar with `ITML` but the above is based on what I understood of it. Feel free to tell me if I'm wrong

Reply:
As currently implemented, `bounds_` should be an ndarray-like of shape (2,).

We should probably update the code to accept list or tuple types for the bounds as well.

Reply:
"these are the ones" -> "these are"